packages/flutter/lib/src/material/text_button.dart

```diff
@@ -320,7 +320,7 @@ class TextButton extends ButtonStyleButton {
   /// * `surfaceTintColor` - null
   /// * `elevation` - 0
   /// * `padding`
-  ///   * `textScaleFactor <= 1` - all(8)
+  ///   * `textScaleFactor <= 1` - lerp(horizontal(8), horizontal(4))
```

Comment:
this doesn't seem to match ![image](https://user-images.githubusercontent.com/6655696/212345446-136144a1-7fdb-4875-960a-4bd6282d8000.png)

Reply:
not a huge fan of encoding spec in comments anyway so don't worry about reproducing every last detail

Reply:
done